### ISSUE TITLE
Renaming Series with the standard naming convention (S{season:00}E{episode:00}) produces error

### ISSUE THREAD

**Issue by Gregg-Williams** [Sonarr/Sonarr#5501]
### Is there an existing issue for this?

- [X] I have searched the existing open and closed issues

### Current Behavior

In the Settings Tab in Media Management , changing the Standard episode format to {Series Title} (S{season:00}E{episode:00}) {Episode Title} produces this error: **Multi Episode: Invalid Format**. This causes the page to issue this warning: **Produces invalid file names**. Consequently, the page will not allow you to save these settings, which is what it should do.

### Expected Behavior

No error on this configuration.

### Steps To Reproduce

1. Go to **Settings**
2. Go to **Media Management**
3. Change **Standard Episode Format** to {Series Title} (S{season:00}E{episode:00})

### Environment

```markdown
- OS: Windows 11 Home / OS Build: 22624.1465
- Sonarr: Version 3.0.10.1567 / .net Version 4.8.0 
- Docker Install: Yes
- Using Reverse Proxy: No
- Browser: Microsoft Edge
```

### What branch are you running?

Main

### Trace Logs?

[sonar.trace.txt](https://github.com/Sonarr/Sonarr/files/11052796/sonar.trace.txt)

### Anything else?

First jpg is without any errors using the ( ) with a space between S{season:00}E{episode:00} as such ( S{season:00}E{episode:00} ).
![No Error](https://user-images.githubusercontent.com/34506343/227252516-d22ba048-976f-4374-b2a4-5145f370a4d1.jpg)

Second jpg is showing the error when using just the ( ) as such (S{season:00}E{episode:00}).
![Error](https://user-images.githubusercontent.com/34506343/227252575-6f462fbc-7a85-4843-800b-ea5567492b47.jpg)

Third jpg is showing the error description after trying to save. this is the correct behavior of the program however and not an error per say, just a consequence of the previous error.
![Error when Saving](https://user-images.githubusercontent.com/34506343/227252623-36a07365-8a39-4d73-ac07-0998a95638c9.jpg)

On a side note:
( S{season:00}E{episode:00}) does not produce an error. 
(S{season:00}E{episode:00} ) does produce an error.

thus it seems as if the encroachment of a character directly in front of the S(season:00} is causing the issue.

**Comment by thezoggy** [comment]
what app do you use that `(S##E##)` is a valid format

**Comment by Gregg-Williams** [comment]
Over the years this has been a standard format. In any event, regardless of whether it is or isn't, this should be viable. At least in my mind, this: **Star Trek - TOS (S01E00) The Cage** is much neater than **Star Trek - TOS - S01E00 - The Cage** 

Peace!

**Comment by markus101** [comment]
It's a niche format, definitely not standard.
How are your existing multi-episode files named with this format?
Have you tried changing the multi-episode style in Sonarr (it's an option below the formats)?

Sonarr needs to be able to parse your naming format and calculate a multi episode format, which it can't do with that format and all multi-episode styles.

**Comment by Gregg-Williams** [comment]
I use the S01E01-03 type format. 

I thought the format standard as that is what I see quite often. Anyhow, if it's not a bug, I apologize for the request. Being that if you type any letter or number immediately in front of S{season:00}, the error comes up, I figured it was a small bug. Funny enough however, upon further investigating, I can type many of the special characters such as $, %, ^, &, ~, |, ", ', :, ;, ?, >, <, etc... without the error exhibiting itself.

Thanks for the response.
